### ISSUE TITLE
Support kubeconfig context

### ISSUE THREAD

**Issue by n0madic** [robscott/kube-capacity#6]
Please add `--context` option like in kubectl

**Comment by rocketraman** [comment]
Yup, this utility tries to parse my configs directly, and fails miserably at it, because KUBECONFIG is a path with multiple files:

echo $KUBECONFIG 
/home/raman/.kube/config-1:/home/raman/.kube/config-2:...

```
/home/raman/.kube/config-1:/home/raman/.kube/config-2:... does not exist - please make sure you have a kubeconfig configured.
```

**Comment by justinbarrick** [comment]
@rocketraman this PR that just landed will help with your issue: https://github.com/robscott/kube-capacity/pull/8

That PR won’t help with the original issue though.

**Comment by robscott** [comment]
@n0madic This issue should be fixed in 0.2.0 (just released) - thanks for the idea!
